<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISP2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayoutConfiguration">
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBoxHiddenLayerCount">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelFunction">
        <property name="text">
         <string>function:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelHiddenLayerNeuronCount">
        <property name="text">
         <string>hidden layer neuron count:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="comboBoxFunction">
        <property name="currentText">
         <string>tanh</string>
        </property>
        <item>
         <property name="text">
          <string>tanh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sigmoid</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxTrainingCycles">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QPushButton" name="pushButtonStart">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelTestsToRun">
        <property name="text">
         <string>tests to run:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="spinBoxHiddenLayerNeuronCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTrainingCycles">
        <property name="text">
         <string>training cycles:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBoxTestsToRun">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelHiddenLayerCount">
        <property name="text">
         <string>hidden layer count:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelLearningRate">
        <property name="text">
         <string>learning rate:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxLearningRate">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutCenter">
      <item>
       <layout class="QGridLayout" name="gridLayoutCurrentImage">
        <item row="3" column="0">
         <widget class="QLabel" name="labelFour">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelSeven">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="labelNine">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelImage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelOne">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelEight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelTwo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelFive">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelExpectedLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelThree">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelSix">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelZero">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Classification</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutBottom">
      <item>
       <widget class="QLabel" name="labelClassificationStatistics">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutProgress">
        <item>
         <widget class="QLabel" name="labelProgress">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
